searx/engines/solidtorrents.py

```diff
@@ -58,7 +56,8 @@ def response(resp):
         title = extract_text(a)
         url = eval_xpath_getindex(a, '@href', 0, None)
         stats = eval_xpath(result, './div//div[contains(@class, "stats")]/div')
```

Comment:
Nit: it is possible to write `stats = eval_xpath_list(result, './div//div[contains(@class, "stats")]/div', min_len=5)` so when the HTML changes, instead of crashing with an `IndexError` few lines later, it crashes with `searx.exceptions.SearxEngineXPathException: ./div//div[contains(@class, "stats")]/div len(xpath_str) < 5`.

Reply:
good point / I fixed it to use min_len

@dalf Thanks a lot for your pragmatic suggestions & comments! .. I'm going to merge this PR ..